### ISSUE TITLE
[Bug] Infinity Miner Freeze

### ISSUE THREAD

**Issue by NephyliaHime** [MeteorDevelopment/meteor-client#1941]
### Describe the bug

Infinity Miner Freeze in 1.18.1 when I want to start the function by clicking on the action.

Freeze are unplayble

### Steps to reproduce

```bash
OS : Windows 11
Minecraft : 1.18.1 - Fabric Loader 0.12.12
Meteor : 0.4.7 - 1399

Use the infinity miner for mine diamonds, you will start to freeze.
```

### Meteor Version

0.4.7-1399

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by ItzVaio** [comment]
Baritone.

**Comment by NephyliaHime** [comment]
> Baritone.

Hi, I don't understand what do you want say, the problem come from Baritone, not Meteorclient ?

**Comment by SWP360** [comment]
what is your iq

**Comment by NephyliaHime** [comment]
> what is your iq

If you have more IQ than me, maybe can you give me a solution before trashtalk

**Comment by sarnthilIsHorny** [comment]
> > what is your iq
> 
> If you have more IQ than me, maybe can you give me a solution before trashtalk

*trash talking

**Comment by SWP360** [comment]
> > what is your iq
> 
> If you have more IQ than me, maybe can you give me a solution before trashtalk

yes, learn what the difference between meteor & baritone is 💋

**Comment by Justgamer101** [comment]
Does this only work for diamonds infinty miner?

**Comment by MineGame159** [comment]
Fixed in 346f5cd9159b80c0d357f754793ec118816f4540.
